code/src/components/MixedQuestions.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+
+const genres = ['Drama', 'Comedy', 'Thriller', 'Mystery', 'Sci-Fi', 'Horror', 'Fantasy', 'Romance', 'Social realism', 'Any']
+const MixedQuestions = ({
+  continent,
+  open,
+  setOpen,
+  genre,
+  setGenre,
+  counter,
+  setCounter
+}) => {
+  const [error, setError] = useState(false);
+
+  const submitOpen = () => {
+    setCounter(counter + 1);
+    if (open === '') {
+      setOpen('not open');
+    }
+  }
+
+  const handleCheckBox = (event) => {
+    setOpen(event.target.value);
+  }
+
+  const submitGenre = () => {
+    if (genre === '') {
+      setError(true)
+    } else {
+      setCounter(counter + 1);
+    }
+  }
+
+  return (
+    <div className="survey-wrapper mixed">
+      {continent ? (
+        <>
+          <h3>
+            <span>
+            Are you basically open to anything?
+            </span>
```

Comment:
In desktop, this adds a scroll bar both horisontally and vertically. On mobile it looks great. This does not bother me so much, it still looks good, but thought I would add a comment about it anyway :)